app/scripts/directives/bindService.js

```diff
@@ -71,6 +71,7 @@ function BindService($scope,
                                         .concat(replicaSets)
                                         .concat(statefulSets);
       ctrl.applications = _.sortByAll(apiObjects, ['metadata.name', 'kind']);
```

Comment:
Maybe something like

bindingType = 'application' | 'secret-only' | 'none'

to avoid confusion around types?